h/paginator.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division
```

Comment:
/me reads https://www.python.org/dev/peps/pep-0238/ - that must have been a fun discussion